common/test/acceptance/pages/lms/progress.py

```diff
@@ -14,7 +14,7 @@ class ProgressPage(CoursePage):
     def is_browser_on_page(self):
         is_present = (
             self.q(css='.course-info').present and
-            self.q(css='#grade-detail-graph').present
+            self.q(css='.grade-detail-graph').present
```

Comment:
This particular query ends up not mattering - the only div with this class also has this id. See note on the other though.